modules/oadp-upgrading-oadp-operator-1-5-0.adoc

```diff
@@ -0,0 +1,27 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="oadp-upgrading-dpa-operator-1-5-0_{context}"]
+= Upgrading the OADP Operator
+
+Use the following procedure when upgrading the {oadp-first} Operator.
+
+[NOTE]
+====
+Do not install OADP 1.5.0 on a Openshift 4.18 cluster.
+====
+
+.Prerequisites
+
+. You have installed the latest {oadp-short} {oadp-version-1-4}.
+. You have backed up your data.
+
+.Procedure
+
+. Upgrade {OCP-short} 4.18 to {OCP-short} 4.19.
++
+[NOTE]
+====
+The {oadp-first} 1.4 is not supported on {OCP-short} 4.19.
+====
+. Change your subscription channel for the OADP Operator from `stable-1.4` to `stable-1.5`.
```

Comment:
```suggestion
. Change your subscription channel for the OADP Operator from `stable-1.4` to `stable`.
```
Starting with 1.5's release on 4.19, single `stable` channel going forward is expected.